core/server/data/schema/fixtures/fixtures.json

```diff
@@ -44,24 +44,28 @@
             "name": "Role",
             "entries": [
                 {
-                    "name":             "Administrator",
-                    "description":      "Administrators"
+                    "name": "Administrator",
+                    "description": "Administrators"
                 },
                 {
-                    "name":             "Editor",
-                    "description":      "Editors"
+                    "name": "Editor",
+                    "description": "Editors"
                 },
                 {
-                    "name":             "Author",
-                    "description":      "Authors"
+                    "name": "Author",
+                    "description": "Authors"
                 },
                 {
-                    "name":             "Contributor",
-                    "description":      "Contributors"
+                    "name": "Contributor",
+                    "description": "Contributors"
                 },
                 {
-                    "name":             "Owner",
-                    "description":      "Blog Owner"
+                    "name": "Owner",
+                    "description": "Blog Owner"
+                },
+                {
+                    "name": "Admin API Client",
```

Comment:
Where is the conversation about this namimg here? I can't find it anymore :(

Reply:
https://github.com/TryGhost/Ghost/pull/9915#discussion_r220881756

Reply:
It was added to the other PR, https://github.com/TryGhost/Ghost/pull/9915#discussion_r220878922

I'm happy to rename it to `Admin Integration`. The thought behind the original `Admin API Client` naming was because it's a role for external clients connecting to the Admin API but seeing as we're calling clients integrations now I think it makes sense to keep the naming consistent.

Reply:
@allouis What do you think?

Reply:
I don't mind too much, but would note that an integration can have both admin and public keys, and "Admin Integration" _kinda_ implies that the integration is only an admin one.

As Kevin said - we were previously going to call them clients, but IIRC you would have had a different client for public and admin access.

Maybe "Admin Integration Key" or "Admin Integration Client" would be totally explicit? Eor update the description so we'll always know what it's for? or just leave it! :shrimp:

Reply:
> and "Admin Integration" kinda implies that the integration is only an admin one

Hmm it's the role for the **admin** integration. The content integration key does not have a role id. Otherwise the role could be called "Integration"?

I def. find it weird having the word "API" inside the role name. Any role is for the API and uses the API. And is also basically a client. I am an editor and i use the API. My role name is "editor" and not "editor client api".

Reply:
> Hmm it's the role for the admin integration.

It's more like the role for the admin key of the integration. Like we have "Integration X" and that integration has a content key and an admin key.

Seems this is just semantics though at this point :man_shrugging: I'm happy with either :relaxed:

Reply:
`Admin Integration` makes the most sense to me at this point and is less ambiguous than `Admin API Client` when placed in context of the general "Integrations" naming.

We may have a `Content Integration` role later once it's possible that members-only content could be accessible via the Content API but there are still quite a few questions around that. Using `Admin Integration` keeps the possibilities open and more clearly defines the level of access that it allows.

Reply:
👍